docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,221 @@
+# Week 10: Deployment
+
+## Agenda
+- Deployment
+	- What is deploying a program
+	- Why do you want to deploy a program
+- Heroku
+	- What is Heroku
+	- Why do we use Heroku
+	- Deploy on Heroku
+- Deploy our application
+	- Deploying Frontend
+	- Deploying Backend
+	- Deploying Database
+	
+## Deployment
+
+### What does it mean to deploy our program?
+
+Deploying a program is the act of making a program publicly accessible on the internet through a URL. A program that is deployed is running on either a remote or local server. A server is basically a machine that is running a program waiting for requests. The server that your program is deployed on is running your program and once the URL that maps to your program is requested, the server is in charge of performing the specified actions that you want it to do based on the request URL that you provided.
+
+### Why deploy a program?
+
+Developers deploy a program in order to publicly showcase their application and allow people to use it. Deploying a program is a great way to showcase what you built or hopefully provide a service to others. Before we deployed our program, everything was running locally on our machine. If you remember, the URL for our program was localhost followed by the port number. However, other people can't access our localhost because, just like the name says, localhost is our local machine server meant just for us. However, once we deploy a program we are given a public URL which others can access and see our application just as we see it in our localhost.
+
+## Heroku
+
+### What is Heroku?
+Heroku is a cloud platform that allows developers to deploy their applications quick, easy, and free. Heroku is configured to handle a variety of programming languages and frameworks such as Java, Javascript, Python, Ruby, etc. For our case, Heroku is a great choice. Heroku works by having developers upload their project and Heroku runs our application for us and connects it to a URL waiting to respond to requests. In traditional cloud deployment platforms, developers have to create a virtual machine with a certain Operating System and hardware capabilities and then deploy their application within the application on the virtual machine and then connect it to a URL. A virtual machine can be envisioned as a computer that is separate from our physical computer. In this case, the virtual machine sits in some lab that the cloud provider your using owns, filled with hundreds of other VM's and you basically "rent" these machines. This is a complicated and long process. Thankfully, Heroku simplifies the process. All developers have to do is connect their code with Heroku and Heroku is in charge of doing the rest. 
```

Comment:
You have a bit of a run-on sentence here. Can you split up this:
> Heroku works by having developers upload their project and Heroku runs our application for us and connects it to a URL waiting to respond to requests.

Maybe simplify this:
>In traditional cloud deployment platforms, developers have to create a virtual machine with a certain Operating System and hardware capabilities and then deploy their application within the application on the virtual machine and then connect it to a URL. A virtual machine can be envisioned as a computer that is separate from our physical computer. In this case, the virtual machine sits in some lab that the cloud provider your using owns, filled with hundreds of other VM's and you basically "rent" these machines. This is a complicated and long process.

to something like:
>In traditional cloud deployment platforms, developers have to manually configure operating systems, deployment methods and procedures, and URL mappings on a remote server somewhere, which can be a long and complicated process. Thankfully, Heroku simplifies the process. All developers have to do is connect their code with Heroku, and Heroku takes care of the rest!